spec/controllers/self_assessments_controller_spec.rb

```diff
@@ -10,8 +10,7 @@
 
   def valid_attributes
     {
-       response: "My Response",
-       learning_assessment: "Learning Response"
+       response: "My Response"
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.